compiler/plc_driver/src/pipelines.rs

```diff
@@ -284,8 +284,16 @@ impl<T: SourceContainer + Sync> AnnotatedProject<T> {
             literals,
             dependencies,
             &self.index,
+            compile_options.online_change,
         )?;
-        code_generator.generate(context, unit, &self.annotations, &self.index, &llvm_index)
+        code_generator.generate(
+            context,
+            unit,
+            &self.annotations,
+            &self.index,
+            &llvm_index,
+            compile_options.online_change,
+        )
```

Comment:
I'm not a fan of duplicating the parameter here, but they do the same for `self.index` and it would require the introduction of one more "context-like" type to `code_generator`. Do you think it's worth it?

Reply:
Hmm, I do feel like it might fit better as a parameter in the constructor, but I can't articulate precisely why! Online change does feel like a piece of information relevant to codegen at the same granularity as the optimization level or the debug level.